crates/turborepo-lib/src/run/summary/execution.rs

```diff
@@ -159,14 +168,23 @@ impl<'a> ExecutionSummary<'a> {
     fn successful(&self) -> usize {
         self.success + self.cached
     }
+
+    fn add_cached(&mut self, source: CacheSource) {
```

Comment:
I don't see this used anywhere so I think it can get removed? In general I think we don't want to be changing the counts on the `ExecutionSummary` and have all modifications happen to `SummaryState`